Generated/FlingAssetPaths.cpp

```diff
@@ -0,0 +1,55 @@
+#include "pch.h"
+#include "FlingPaths.h"
+
+namespace Fling
```

Comment:
Why is the `Generated` folder in here? Anything you modified in this file you should have done in `FlingAssetPaths.cpp.in` because this one is generated by cmake (see the absolute paths)